docs/source/integrations/databricks_deployment_workflow.md

```diff
@@ -0,0 +1,316 @@
+Because you are no longer using the default entry-point for Kedro, you will not be able to run your project with the options it usually provides. Instead, the `databricks_run` entry point in the above code and in the PySpark Iris starter contains a simple implementation of two options:
+- `--env`: specifies a configuration environment to load for your run.
+- `--conf-source`: specifies the location of the `conf/` directory to use with your Kedro project.
+```
+
+### Plan your project's output
+
+You will not be able to directly see your project's log output when it is run as a Databricks job. You will need to decide on and implement a strategy for storing your project's results. Some examples are:
+
+- Write the result of a data engineering pipeline to a managed delta table.
+- Upload metrics from a machine learning model to MLflow.
+- Save a dataset as a CSV file in DBFS.
+
+In this guide, your project will save its logs to a location in DBFS.
+
+- **Create a directory to save your logs into on DBFS**: at the command line in your local environment, run the following Databricks CLI command:
+
+```bash
+databricks fs mkdirs dbfs:/FileStore/iris-databricks/logs
+```
+
+This command creates a directory on DBFS (`/dbfs/FileStore/iris-databricks/logs`) where your project can save its logs.
+
+**The `databricks` environment of the PySpark Iris starter is already configured to save logs in this location.** The configuration used to achieve this is part of the [`databricks` environment](../configuration/configuration_basics.md#configuration-environments) defined for PySpark Iris and can be found in the `conf/databricks/logging.yml` file.
+
+### Package your project
+
+To package your Kedro project for deployment on Databricks, you must create a Wheel (`.whl`) file, which is a binary distribution of your project. In the root directory of your Kedro projec   t, run the following command:
+
+```bash
+kedro package
+```
+
+This command generates a `.whl` file in the `dist` directory within your project's root directory.
+
+### Upload project data and configuration to DBFS
+
+Your packaged Kedro project needs access to data and configuration in order to run. Therefore, you will need to upload your project's data and configuration to an accessible location. In this guide, we will store the data on the Databricks File System (DBFS).
+
+The PySpark Iris starter contains an environment that is set up to access data stored in DBFS (`conf/databricks`). To learn more about environments in Kedro configuration, see the [configuration documentation](../configuration/configuration_basics.md#configuration-environments). When you run your project on Databricks, you will pass the name of this environment as an option, as well as the path to the `conf/` directory on DBFS.
+
+There are several ways to upload data to DBFS. In this guide, it is recommended to use [Databricks CLI](https://docs.databricks.com/dev-tools/cli/dbfs-cli.html) because of the convenience it offers.
```

Comment:
```suggestion
There are several ways to upload data to DBFS but we recommend that [Databricks CLI](https://docs.databricks.com/dev-tools/cli/dbfs-cli.html) because of the convenience it offers.
```

Reply:
Again, you could list the other options of the "several ways" you mention.

Reply:
I've mentioned two other ways, plus links. There are one or two more I could think of, but I think it would be too confusing to add them.